tsconfig.json

```diff
@@ -12,8 +12,9 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/pages/_document.js"],
```

Comment:
Is this needed? 👀

Reply:
Next's lint settings require the _document to be .js, because they never updated their lint docs to acknowledge .tsx

Reply:
I almost rather skip lint then lol 😆